pkg/webservice/handlers_test.go

```diff
@@ -1405,7 +1422,7 @@ func TestFullStateDumpPath(t *testing.T) {
 }
 
 func TestSpecificUserAndGroupResourceUsage(t *testing.T) {
-	prepareUserAndGroupContext(t)
+	prepareUserAndGroupContext(t, configDefault)
```

Comment:
(this is not related to this PR)
it seems the `configDefault` does not include any group/user, so all asserts in this test case are related to "nonexistent" case, right?

Reply:
> it seems the configDefault does not include any group/user, so all asserts in this test case are related to "nonexistent" case, right?

You're correct, except for the test case under comment `// Test existed user query`. I believe the test case mentioned in that comment is missing an assertion to verify the existence of the user resource `testuser`.

Reply:
That is unrelated to this PR, so I file ticket (https://issues.apache.org/jira/browse/YUNIKORN-2272) to trace it.

Reply:
Thanks, already take over the ticket, will come up with another pr ASAP.